packages/nextjs/package.json

```diff
@@ -17,6 +17,7 @@
     "access": "public"
   },
   "dependencies": {
+    "@babel/parser": "^7.18.10",
```

Comment:
Are we using this?

Reply:
Yes. See above.

> - Parsing of user code into an AST (and subsequent traversal of that AST) is done using `@babel/parser` and `jscodeshift`. (`@babel/parser` is used directly because `jscodeshift` neither exposes its parsers nor provides an option for specifying a parser by name (at least when used programmatically). Therefore, in order to use a jsx parser for JS pages and a tsx parser for TS pages, we need to supply them ourselves.)